nomad/fsm.go

```diff
@@ -2249,7 +2249,8 @@ func (f *FSMFilter) Include(item interface{}) bool {
 	return true
 }
 
-func (n *nomadFSM) applyVariableOperation(msgType structs.MessageType, buf []byte, index uint64) interface{} {
+func (n *nomadFSM) applyVariableOperation(msgType structs.MessageType, buf []byte,
+	index uint64) interface{} {
```

Comment:
As long as we're touching it:

```suggestion
	index uint64) any {
```